### DISCUSSION TITLE
Update this plugin?

### DISCUSSION THREAD

**Discussion by pixelstock** [Pryaxis/TShock#1760] [Q&A]
i use tshock pre release 1.4.2

https://tshock.co/xf/index.php?resources/autoregister.234/

Could someone adapt it for me?
I need it and I don't know how to do it

**Comment by fernandofig** [comment]
just pinging to let you know I got it to compile after a couple of trivial changes. I haven't tested it, but the changes required and the plugin as whole seems simple enough that I don't think it'll have any issues.

I'm not the original author though, so I'm not sure how to go about to publishing the binary. Since as part of tshock's site changes a central plugin repository will no longer exist, I suppose I'd be expected to fork the original and release it on github, but that kinda feels like too much for me, I mean, I don't want anybody expecting me to support or maintain it, it's just a one-off thing I did as I ported a few other plugins I needed myself.
